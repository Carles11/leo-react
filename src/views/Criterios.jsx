import React from 'react'
import Helmet from 'react-helmet'

const Criterios = (props) => {
  const { DIC } = props

  return (
    <div className="app-content pb2rem mb2rem">
      <Helmet
        title={DIC.NAV_CRITERIOS}
        meta={[
				  { name: 'description', content: `${DIC.NAV_CRITERIOS}` },
				  { property: 'og:title', content: `${DIC.NAV_CRITERIOS}` },
        ]}
      />
      <header>
        <h1 className="tit-header mb2rem">
          {DIC.NAV_CRITERIOS}
        </h1>
      </header>
      <article className="app-section-width">
        <p className="txt">
          <strong className="txt-highlight">
Criterios de lectura para el jurado:&nbsp;
          </strong>
					La "calidad" de la lectura se determinará según los siguientes dos criterios:
          <ul>
            <li>
Pronunciación / entonación
            </li>
            <li>
Comprensión / interpretación
            </li>
          </ul>
					La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.

        </p>
        <p className="txt">
          <strong className="txt-highlight">
Pronunciación/entonación:&nbsp;
          </strong>
					pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta
					deficiencias en la pronunciación de la "r/rr"), acentuación correcta y velocidad de
					lectura. No se penalizará la autocorrección.

        </p>
        <p className="txt">
          <strong className="txt-highlight">
Comprensión/interpretación:&nbsp;
          </strong>
					el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de
					frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto
					ambiental de la lectura.

        </p>
      </article>
    </div>
  )
}

export default Criterios
