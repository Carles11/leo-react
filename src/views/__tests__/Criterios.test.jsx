import React from 'react'
import { shallow } from 'enzyme'
import { Unwrapped as UnwrappedCriterios } from '../Criterios'
import getDictiorary from '../../utils/dictionary'

const properties = {
  DIC: getDictiorary(),
}

describe('<Criterios />', () => {
  it('should render itself', () => {
    const component = shallow(<UnwrappedCriterios {...properties} />)
    expect(component).toMatchSnapshot()
  })
})
