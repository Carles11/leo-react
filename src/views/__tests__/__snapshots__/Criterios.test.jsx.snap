// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Criterios /> should render itself 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Criterios
    DIC={
        Object {
            "CONCURSO": "El Concurso",
            "CONCURSO_EDICION": "¿Cuándo es la próxima edición?",
            "CONCURSO_EDICION_TXT": "La Final de la octava convocatoria del Concurso Escolar de Lectura en Español ‟Leo, leo… ¿Qué lees?” tendrá lugar en el Salón de Actos de la sede del Instituto Cervantes de Fráncfort el día 16 de mayo de 2019 en la sede de Fráncfort del Instituto Cervantes. La fecha límite para la inscripción de los colegios participantes será el 30 de noviembre de 2018.",
            "CONCURSO_ORGANIZA": "¿Quién lo organiza?",
            "CONCURSO_ORGANIZA_TXT": "Siguiendo la larga tradición de los concursos de lectura en Alemania el Concurso Escolar de Lectura en Español ‟Leo, leo… ¿Qué lees ?” nació como propuesta educativa en el colegio Hessenwaldschule (Weiterstadt) para dar respuesta a un vacío de iniciativas que reforzaran el aprendizaje de la lengua española.",
            "CONCURSO_PARTICIPAR": "¿Quién puede participar?",
            "CONCURSO_PARTICIPAR_TXT": "Todos los alumnos de secundario en la enseñanza reglada del sistema educativo alemán con español como segunda y tercera lengua pueden participar en el concurso, que se desarrolla en cuatro categorías de A1 a B2, según el Marco Común Europeo de Referencia para las lenguas (MCER).",
            "CONCURSO_PATROCINA": "¿Quién nos patrocina?",
            "CONCURSO_PATROCINA_TXT": "En su octavo año de celebración consecutiva cuenta con el apoyo institucional de la Consejería de Educación de la Embajada de España en Alemania, del Instituto Cervantes de Fráncfort, del Consulado General de España en Fráncfort, del Consulado General de México en Fráncfort, del Consulado General de Colombia en Fráncfort, del Consulado General de Chile en Fráncfort y de la Asociación Alemana de Profesores de Español en Hesse (DSV). Asímismo con el patrocinio de las editoriales Grupo SM, Schulverlag Klett, Cornelsen Verlag y Edinumen.",
            "CONCURSO_TXT": "La lectura nos ayuda a comunicarnos con nuestro entorno y los concursos motivan a alumnos y profesores de todos los ámbitos de la enseñanza a conseguir nuevas metas. En este contexto surgió en 2012 la idea de poner en marcha un concurso de lectura en español en Hesse. Tras una fase de prueba se ha ido consolidando la iniciativa obteniendo en 2015 el apoyo del Kultusministerium de Hesse y entrando a formar parte de la red oficial de concursos escolares en esta Comunidad Autónoma. La trayectoria del concurso desde 2012 ha demostrado la gran acogida del concurso por parte de los centros educativos.Esto se ha debido especialmente a la fácil organización del concurso en la clase de español. Todos los alumnos de secundario en la enseñanza reglada del sistema educativo alemán con español como segunda y tercera lengua pueden participar en el concurso, que se desarrolla en cuatro categorías de A1 a B2, según el Marco Común Europeo de Referencia para las lenguas(MCER).",
            "DESCRIPTION": "Concurso de lectura en español",
            "ERROR_EMAIL": "debe ser un correo electronico válido.",
            "ERROR_REQUIRED": "es un campo obligatorio.",
            "ERROR_TEXT": "debe ser mas de 6 caracteres",
            "FORM_ADDRESS": "Dirección",
            "FORM_CATEGORIES": "Selecciona las categorias del concurso en las que participará el colegio:",
            "FORM_CONTACT": "Persona de contacto",
            "FORM_COURSE": "Curso",
            "FORM_MAIL": "Correo electrónico",
            "FORM_PHONE": "Teléfono",
            "FORM_SCHOOL_NAME": "Nombre del colegio",
            "FORM_SUBTITLE": "Registra tu colegio",
            "FORM_TITLE": "Formulario de inscripción",
            "HEADER_MAIN": "Leo, leo... ¿qué lees?",
            "MORE_INFO": "Más información",
            "MSG_SCHOOL_CREATED": "Gracias por inscribirse al concurso de lectura en español \\"Leo, leo... ¿qué lees?\\". En breve nos pondremos en contacto con usted.",
            "MSG_SCHOOL_ERROR": "Lamentablemente no se pudo enviar el formulario, inténtelo más tarde.",
            "NAME": "Leo, leo",
            "NAV_BASES": "Bases del concurso",
            "NAV_CERTIFICADOS": "Certificados",
            "NAV_COLEGIOS": "Colegios inscritos",
            "NAV_CONCURSO": "Concurso",
            "NAV_CRITERIOS": "Criterios de lectura",
            "NAV_GALERIA": "Galeria",
            "NAV_INFO": "Información para profesores",
            "NAV_INSCRIPCION": "Inscripción",
            "NAV_TEXTOS": "Textos de lectura",
            "NOTFOUND": "No se ha encontrado la página.",
            "NOTFOUND_TXT": "Lo sentimos, pero probablemente el link que ha seguido está roto o se ha eliminado la página.",
            "SPONSOR": "Cooperación y patrocinio: ",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          meta={
                    Array [
                              Object {
                                "content": "Criterios de lectura",
                                "name": "description",
                              },
                              Object {
                                "content": "Criterios de lectura",
                                "property": "og:title",
                              },
                            ]
          }
          title="Criterios de lectura"
/>,
        <header>
          <h1
                    className="tit-header mb2rem"
          >
                    Criterios de lectura
          </h1>
</header>,
        <article
          className="app-section-width"
>
          <p
                    className="txt"
          >
                    <strong
                              className="txt-highlight"
                    >
                              Criterios de lectura para el jurado: 
                    </strong>
                    La "calidad" de la lectura se determinará según los siguientes dos criterios:
          </p>
          <ul>
                    <li>
                              Pronunciación / entonación
                    </li>
                    <li>
                              Comprensión / interpretación
                    </li>
          </ul>
          <p
                    className="txt"
          >
                    La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.
          </p>
          <p
                    className="txt"
          >
                    <strong
                              className="txt-highlight"
                    >
                              Pronunciación/entonación: 
                    </strong>
                    pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta deficiencias en la pronunciación de la "r/rr"), acentuación correcta y velocidad de lectura. No se penalizará la autocorrección.
          </p>
          <p
                    className="txt"
          >
                    <strong
                              className="txt-highlight"
                    >
                              Comprensión/interpretación: 
                    </strong>
                    el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto ambiental de la lectura.
          </p>
</article>,
      ],
      "className": "app-content pb2rem mb2rem",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "meta": Array [
            Object {
              "content": "Criterios de lectura",
              "name": "description",
            },
            Object {
              "content": "Criterios de lectura",
              "property": "og:title",
            },
          ],
          "title": "Criterios de lectura",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            className="tit-header mb2rem"
>
            Criterios de lectura
</h1>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Criterios de lectura",
            "className": "tit-header mb2rem",
          },
          "ref": null,
          "rendered": "Criterios de lectura",
          "type": "h1",
        },
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="txt"
>
              <strong
                            className="txt-highlight"
              >
                            Criterios de lectura para el jurado: 
              </strong>
              La "calidad" de la lectura se determinará según los siguientes dos criterios:
</p>,
            <ul>
              <li>
                            Pronunciación / entonación
              </li>
              <li>
                            Comprensión / interpretación
              </li>
</ul>,
            <p
              className="txt"
>
              La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.
</p>,
            <p
              className="txt"
>
              <strong
                            className="txt-highlight"
              >
                            Pronunciación/entonación: 
              </strong>
              pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta deficiencias en la pronunciación de la "r/rr"), acentuación correcta y velocidad de lectura. No se penalizará la autocorrección.
</p>,
            <p
              className="txt"
>
              <strong
                            className="txt-highlight"
              >
                            Comprensión/interpretación: 
              </strong>
              el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto ambiental de la lectura.
</p>,
          ],
          "className": "app-section-width",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <strong
                  className="txt-highlight"
>
                  Criterios de lectura para el jurado: 
</strong>,
                "La \\"calidad\\" de la lectura se determinará según los siguientes dos criterios:",
              ],
              "className": "txt",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Criterios de lectura para el jurado: ",
                  "className": "txt-highlight",
                },
                "ref": null,
                "rendered": "Criterios de lectura para el jurado: ",
                "type": "strong",
              },
              "La \\"calidad\\" de la lectura se determinará según los siguientes dos criterios:",
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  Pronunciación / entonación
</li>,
                <li>
                  Comprensión / interpretación
</li>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Pronunciación / entonación",
                },
                "ref": null,
                "rendered": "Pronunciación / entonación",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Comprensión / interpretación",
                },
                "ref": null,
                "rendered": "Comprensión / interpretación",
                "type": "li",
              },
            ],
            "type": "ul",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.",
              "className": "txt",
            },
            "ref": null,
            "rendered": "La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <strong
                  className="txt-highlight"
>
                  Pronunciación/entonación: 
</strong>,
                "pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta deficiencias en la pronunciación de la \\"r/rr\\"), acentuación correcta y velocidad de lectura. No se penalizará la autocorrección.",
              ],
              "className": "txt",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Pronunciación/entonación: ",
                  "className": "txt-highlight",
                },
                "ref": null,
                "rendered": "Pronunciación/entonación: ",
                "type": "strong",
              },
              "pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta deficiencias en la pronunciación de la \\"r/rr\\"), acentuación correcta y velocidad de lectura. No se penalizará la autocorrección.",
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <strong
                  className="txt-highlight"
>
                  Comprensión/interpretación: 
</strong>,
                "el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto ambiental de la lectura.",
              ],
              "className": "txt",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Comprensión/interpretación: ",
                  "className": "txt-highlight",
                },
                "ref": null,
                "rendered": "Comprensión/interpretación: ",
                "type": "strong",
              },
              "el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto ambiental de la lectura.",
            ],
            "type": "p",
          },
        ],
        "type": "article",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            meta={
                        Array [
                                    Object {
                                      "content": "Criterios de lectura",
                                      "name": "description",
                                    },
                                    Object {
                                      "content": "Criterios de lectura",
                                      "property": "og:title",
                                    },
                                  ]
            }
            title="Criterios de lectura"
/>,
          <header>
            <h1
                        className="tit-header mb2rem"
            >
                        Criterios de lectura
            </h1>
</header>,
          <article
            className="app-section-width"
>
            <p
                        className="txt"
            >
                        <strong
                                    className="txt-highlight"
                        >
                                    Criterios de lectura para el jurado: 
                        </strong>
                        La "calidad" de la lectura se determinará según los siguientes dos criterios:
            </p>
            <ul>
                        <li>
                                    Pronunciación / entonación
                        </li>
                        <li>
                                    Comprensión / interpretación
                        </li>
            </ul>
            <p
                        className="txt"
            >
                        La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.
            </p>
            <p
                        className="txt"
            >
                        <strong
                                    className="txt-highlight"
                        >
                                    Pronunciación/entonación: 
                        </strong>
                        pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta deficiencias en la pronunciación de la "r/rr"), acentuación correcta y velocidad de lectura. No se penalizará la autocorrección.
            </p>
            <p
                        className="txt"
            >
                        <strong
                                    className="txt-highlight"
                        >
                                    Comprensión/interpretación: 
                        </strong>
                        el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto ambiental de la lectura.
            </p>
</article>,
        ],
        "className": "app-content pb2rem mb2rem",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "meta": Array [
              Object {
                "content": "Criterios de lectura",
                "name": "description",
              },
              Object {
                "content": "Criterios de lectura",
                "property": "og:title",
              },
            ],
            "title": "Criterios de lectura",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="tit-header mb2rem"
>
              Criterios de lectura
</h1>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Criterios de lectura",
              "className": "tit-header mb2rem",
            },
            "ref": null,
            "rendered": "Criterios de lectura",
            "type": "h1",
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="txt"
>
                <strong
                                className="txt-highlight"
                >
                                Criterios de lectura para el jurado: 
                </strong>
                La "calidad" de la lectura se determinará según los siguientes dos criterios:
</p>,
              <ul>
                <li>
                                Pronunciación / entonación
                </li>
                <li>
                                Comprensión / interpretación
                </li>
</ul>,
              <p
                className="txt"
>
                La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.
</p>,
              <p
                className="txt"
>
                <strong
                                className="txt-highlight"
                >
                                Pronunciación/entonación: 
                </strong>
                pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta deficiencias en la pronunciación de la "r/rr"), acentuación correcta y velocidad de lectura. No se penalizará la autocorrección.
</p>,
              <p
                className="txt"
>
                <strong
                                className="txt-highlight"
                >
                                Comprensión/interpretación: 
                </strong>
                el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto ambiental de la lectura.
</p>,
            ],
            "className": "app-section-width",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <strong
                    className="txt-highlight"
>
                    Criterios de lectura para el jurado: 
</strong>,
                  "La \\"calidad\\" de la lectura se determinará según los siguientes dos criterios:",
                ],
                "className": "txt",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Criterios de lectura para el jurado: ",
                    "className": "txt-highlight",
                  },
                  "ref": null,
                  "rendered": "Criterios de lectura para el jurado: ",
                  "type": "strong",
                },
                "La \\"calidad\\" de la lectura se determinará según los siguientes dos criterios:",
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    Pronunciación / entonación
</li>,
                  <li>
                    Comprensión / interpretación
</li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Pronunciación / entonación",
                  },
                  "ref": null,
                  "rendered": "Pronunciación / entonación",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Comprensión / interpretación",
                  },
                  "ref": null,
                  "rendered": "Comprensión / interpretación",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.",
                "className": "txt",
              },
              "ref": null,
              "rendered": "La puntuación será de 1 a 5, para cada una de las tres categorías, siendo 5 el máximo.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <strong
                    className="txt-highlight"
>
                    Pronunciación/entonación: 
</strong>,
                  "pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta deficiencias en la pronunciación de la \\"r/rr\\"), acentuación correcta y velocidad de lectura. No se penalizará la autocorrección.",
                ],
                "className": "txt",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Pronunciación/entonación: ",
                    "className": "txt-highlight",
                  },
                  "ref": null,
                  "rendered": "Pronunciación/entonación: ",
                  "type": "strong",
                },
                "pronunciación clara y correcta (en los niveles A1 y A2 no se tendrán en cuenta deficiencias en la pronunciación de la \\"r/rr\\"), acentuación correcta y velocidad de lectura. No se penalizará la autocorrección.",
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <strong
                    className="txt-highlight"
>
                    Comprensión/interpretación: 
</strong>,
                  "el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto ambiental de la lectura.",
                ],
                "className": "txt",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Comprensión/interpretación: ",
                    "className": "txt-highlight",
                  },
                  "ref": null,
                  "rendered": "Comprensión/interpretación: ",
                  "type": "strong",
                },
                "el candidato demuestra la comprensión de lo leído a través de la segmentación correcta de frases y el uso de recursos prosódicos. Así mismo consigue reflejar el tono y el contexto ambiental de la lectura.",
              ],
              "type": "p",
            },
          ],
          "type": "article",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
